src/main/java/wraith/fabricaeexnihilo/util/EntrypointHelper.java

```diff
@@ -119,7 +112,19 @@ public Block registerCrucible(String name, boolean isFireproof, AbstractBlock.Se
         @Override
         public Block registerCrushedBlock(String name, AbstractBlock.Settings settings) {
             var id = id(name, null, null);
-            var block = ModBlocks.CRUSHED.computeIfAbsent(id, __ -> new FallingBlock(settings));
+            class DummyFallingBlock extends FallingBlock {
+                private final MapCodec<? extends FallingBlock> CODEC = createCodec(DummyFallingBlock::new);
+
+                public DummyFallingBlock(Settings settings) {
+                    super(settings);
+                }
+
+                @Override
+                protected MapCodec<? extends FallingBlock> getCodec() {
+                    return CODEC;
+                }
+            }
```

Comment:
It can't be made anonymous due to the codec requirement(if it is then let me know), but I can pull this out into a separate class file and put it in e.g. util.